### PR TITLE
Use namespace varible for package folder instead of hardcoding it

### PR DESCRIPTION
**Pull request by vladbarosan**

### CODE REVIEW

File: specification/compute/resource-manager/readme.go.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Please also specify `--go-sdk-folder=<path to the root directory of your azure-s
 
 ``` yaml $(tag)=='package-skus-2017-09' && $(go)
 namespace: skus
-output-folder: $(go-sdk-folder)/services/compute/mgmt/2017-09-01/skus
+output-folder: $(go-sdk-folder)/services/$(namespace)/mgmt/2017-09-01/skus
```
Comment (jhendrixMSFT) [inline specification/compute/resource-manager/readme.go.md]: This one is wrong.

File: specification/mediaservices/resource-manager/readme.go.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ These settings apply only when `--tag=package-2015-10 --go` is specified on the
 Please also specify the `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
 ``` yaml $(tag) == 'package-2015-10' && $(go)
-output-folder: $(go-sdk-folder)/services/mediaservices/mgmt/2015-10-01/media
+output-folder: $(go-sdk-folder)/services/$(namespace)services/mgmt/2015-10-01/$(namespace)
```
Comment (jhendrixMSFT) [inline specification/mediaservices/resource-manager/readme.go.md]: Another "kinda right", applied to entire file.

**Review by jhendrixMSFT — CHANGES_REQUESTED** [review]
Found a few bugs.

File: specification/storsimple8000series/resource-manager/readme.go.md [hunk 3]
```diff
@@ -22,5 +22,5 @@ These settings apply only when `--tag=package-2017-06 --go` is specified on the
 Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
 ``` yaml $(tag) == 'package-2017-06' && $(go)
-output-folder: $(go-sdk-folder)/services/storsimple8000series/mgmt/2017-06-01/storsimple
+output-folder: $(go-sdk-folder)/services/$(namespace)8000series/mgmt/2017-06-01/$(namespace)
```
Comment (vladbarosan) [inline specification/storsimple8000series/resource-manager/readme.go.md]: >8000series [](start = 53, length = 10)

here also

File: specification/storSimple1200Series/resource-manager/readme.go.md [hunk 4]
```diff
@@ -22,5 +22,5 @@ These settings apply only when `--tag=package-2016-10 --go` is specified on the
 Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
 ``` yaml $(tag) == 'package-2016-10' && $(go)
-output-folder: $(go-sdk-folder)/services/storsimple1200series/mgmt/2016-10-01/storsimple
+output-folder: $(go-sdk-folder)/services/$(namespace)1200series/mgmt/2016-10-01/$(namespace)
```
Comment (jhendrixMSFT) [inline specification/storSimple1200Series/resource-manager/readme.go.md]: And here.

File: specification/storSimple1200Series/resource-manager/readme.go.md [hunk 5]
```diff
@@ -21,6 +21,6 @@ batch:
 These settings apply only when `--tag=package-2016-10 --go` is specified on the command line.
 Please also specify `--go-sdk-folder=<path to the root directory of your azure-sdk-for-go clone>`.
 
-``` yaml $(tag) == 'package-2016-10' && $(go)
-output-folder: $(go-sdk-folder)/services/$(namespace)1200series/mgmt/2016-10-01/$(namespace)
+```yaml $(tag) == 'package-2016-10' && $(go)
+output-folder: $(go-sdk-folder)/services/$storsimple1200series/mgmt/2016-10-01/$(namespace)
```
Comment (jhendrixMSFT) [inline specification/storSimple1200Series/resource-manager/readme.go.md]: Extra '$' character.

**Review by jhendrixMSFT — APPROVED** [review]

**Comment by jhendrixMSFT** [comment]
@fearthecowboy can you merge?  the tools choke on the large number of commits :(

**Pull request merged**